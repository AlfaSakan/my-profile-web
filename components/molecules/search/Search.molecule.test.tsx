import renderer from 'react-test-renderer';
import Search from './Search.molecule';

describe('Search component test', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Search />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
