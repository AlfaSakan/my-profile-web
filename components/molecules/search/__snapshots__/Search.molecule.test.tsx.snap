// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component test should render correctly 1`] = `
<div
  className="flex p-2 pl-3 bg-white w-full rounded-md items-center undefined"
>
  <div
    className="mr-4"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root"
      focusable="false"
      style={
        Object {
          "fill": "#2C3333",
          "fontSize": 20,
        }
      }
      viewBox="0 0 24 24"
    >
      <path
        d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
      />
    </svg>
  </div>
  <input
    className="w-full focus:outline-none bg-transparent h-full text-dark"
    onChange={[Function]}
    placeholder="Search or start new chat"
    type="text"
  />
</div>
`;
